draft-ietf-httpbis-header-structure.md

```diff
@@ -498,13 +491,14 @@ Given an integer as input_integer:
 
 Given a float as input_float:
 
-0. If input_float is not a IEEE 754 double precision number, fail serialisation.
-1. Let output be an empty string.
-2. If input_float is less than (but not equal to) 0, append "-" to output.
-3. Append input_float's integer component represented in base 10 using only decimal digits to output; if it is zero, append "0".
-4. Append "." to output.
-5. Append input_float's decimal component represented in base 10 using only decimal digits to output; if it is zero, append "0".
-6. Return output.
+0. If input_float's fractional part has more than six digits of precision, fail serialisation.
+1. If the number of digits of precision in input_float's fractional part plus those in its integer part add to more than fifteen digits, fail serialisation.
```

Comment:
Not sure I agree with this one, what if the value is 1/3 ?

Reply:
Perhaps something like:

> Append up to six digits of input_float's decimal component...

Reply:
1/3 doesn't match "integers with a fractional part, which can be at most six digits of precision." This isn't the algorithm for serialising an IEEE-754 float, nor an arbitrary rational number.

Although maybe guidance for converting those to an SH-float data object could be useful.